Test.c

```diff
@@ -4,11 +4,139 @@
 // Ayisha Sameera,GET.
 // ------------------------------------------------------------------
 // Test.c
-// Program on branch main
+// Program on branch A5
 // ------------------------------------------------------------------------------------------------
-
+#pragma warning(disable:4996)
+#include "Header.h"
 #include <stdio.h>
+#include <stdbool.h>
+#include <malloc.h>
+#include <string.h>
+#include <stdlib.h>
+
+typedef struct {
+   int error;            // Stores the error code.
+   int num;              // Stores the input value.
+}Type;
+
+/// <summary>Windows OS specific command to clear console screen.</summary>
+static void ClearScreen () {
+#ifdef _WIN32
+   system ("cls");
+#endif
+}
+
+/// <summary>Checks if the array is in ascending order.</summary>
+bool Check (int arr[], int length) {

```

Comment:
'Check' is not a proper name for method, Method name should self-explain the process